LayoutTestBase/Config/LYTConfig.m

```diff
@@ -12,7 +12,7 @@
 
 @implementation LYTConfig
 
-NSUInteger const LYTSaveUnlimitedSnapshotsPerMethod = -1;
+NSUInteger const LYTSaveUnlimitedSnapshotsPerMethod = (NSUInteger)-1;
```

Comment:
NSUInteger -1 ??

Reply:
AKA NSIntegerMax - this just needs to be a distinct value, but there's no reason it can't be an int.